### ISSUE TITLE
Bad ftdetect causes this plugin to load on any filetype

### ISSUE THREAD

**Issue by oblitum** [JuliaEditorSupport/julia-vim#261]
The ftdetect procedure being used in this plugin must not be correct because it's causing the plugin to execute anytime vim or neovim is started, even when not opening Julia files. It causes `LaTeXtoUnicode#Refresh()`, which slow downs editor startuptime.

**Comment by carlobaldassi** [comment]
This is intended, to allow L2U functionality outside of julia files (#33).
I compared the startup times (using the `--startup` option) of "vanilla" neovim (empty init.vim) versus an init file with only julia-vim plugin. On average, I measured an additional delay of about 20ms.

**Comment by oblitum** [comment]
It should be behind a flag with default _off_. 20ms is a considerable amount of time when most of time vim is used for general editing remotely related to this feature, and it's should be larger on lower end machines. Plugins is general load optionally so that their loading time don't stack up, as people may in general load hundreds of them.

**Comment by oblitum** [comment]
Besides the reported issue, the other one that made me actually report this, is that once profiling my startup time and learning that the julia-vim was responsible for slower startup while no julia file was being edited, I tried to set it as an optional plugin on my package manager. Some Vim package managers offer the option of having a listed plugin as explicitly optional to load, and I did that for julia-vim to load only when filetype julia gets detected, but this ended up causing the plugin manager (packer.nvim) to give errors due to missing functions. I'm not sure what happened but I have dozen of opt plugin and none have hit such issue.
